config/galaxy.yml.sample

```diff
@@ -52,8 +52,7 @@ uwsgi:
   # --daemon and/or production deployments.
   master: false
 
-  # Path to the application's Python virtual environment. If using Conda for
-  # Galaxy's framework dependencies (not tools!), do not set this.
+  # Path to the application's Python virtual environment.
```

Comment:
This is reverting the change introduced with commit https://github.com/galaxyproject/galaxy/commit/a88e19064a38415d867f4c118dd6938d8e368632 , which should instead have been applied also to `lib/galaxy/webapps/config_manage.py` .

Reply:
@nsoranzo I did a config rebuild, that's what overwrote that change I think.  I'll apply that change correctly, I guess, and then rebuild again in this branch.

Reply:
Done.  Thanks for noticing this, @nsoranzo!